mainapp/static/styles.css

```diff
@@ -9,4 +9,4 @@ footer {
     height: 30px;
     line-height: 30px;
     background-color: #f5f5f5;
-}
\ No newline at end of file
+}
```

Comment:
I love this change 👍

Reply:
hahahaha, a critical update to the project to be sure!